baton-maven-plugin/src/main/java/org/technologybrewery/baton/util/FileUtils.java

```diff
@@ -0,0 +1,204 @@
+package org.technologybrewery.baton.util;
+
+import org.apache.commons.io.IOUtils;
+import org.codehaus.plexus.util.StringUtils;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class FileUtils {
+
+    /**
+     *
```

Comment:
S: appreciate the input/output comments, but let's put on for the method too so it's fully javadoc conformant.